framework/widgets/ActiveForm.php

```diff
@@ -30,12 +30,12 @@ class ActiveForm extends Widget
      * @event ActiveFieldEvent an event raised right before rendering an ActiveField.
      * @since 2.1.0
      */
-    const EVENT_BEFORE_FIELD_RENDER = 'beforeFieldRender';
+    const EVENT_BEFORE_FIELD_RENDER = 'widget.form.field.render.before';
     /**
      * @event ActionEvent an event raised right after rendering an ActiveField.
      * @since 2.1.0
      */
-    const EVENT_AFTER_FIELD_RENDER = 'afterFieldRender';
+    const EVENT_AFTER_FIELD_RENDER = 'widget.form.field.render.after';
```

Comment:
Should it be reflected that it's activeform?

Reply:
Should event names which are in classes, which are likely to be moved to separate repos after 2.0.14, contain a special naming convention?

Reply:
Not really.